workspaces/cli-scripts/src/errors.ts

```diff
@@ -29,11 +29,13 @@ export function trackWithSentry({
   const hostPlatform = process.platform;
   const hostCpuCount = Os.cpus().length.toString();
   const hostMemorySize = Os.totalmem().toString();
+  const hostRelease = Os.release();
```

Comment:
I forgot we've duplicated this code in `cli-server/src/errors.ts` as well - would you mind applying it there too?

Reply:
Looks like that's already done, awesome!